app/assets/stylesheets/pages/_milestones.scss

```diff
@@ -212,8 +215,8 @@ time {
 .dropdownn:hover .dropdown-content {display: block;}
 
 .btn-medium {
```

Comment:
you have to be page specific with items.. .btn-medium will target every button throughout the whole application with that class.